lib/stdlib/src/math.erl

```diff
@@ -134,8 +134,31 @@ tan(_) ->
 tanh(_) ->
     erlang:nif_error(undef).
 
+-spec ceilf(X) -> float() when
+      X :: number().
+ceilf(_) ->
+    erlang:nif_error(undef).
+
+-spec floorf(X) -> float() when
+      X :: number().
+floorf(_) ->
+    erlang:nif_error(undef).
+
 %%% End of BIFs
 
+-spec ceil(X) -> integer() when
```

Comment:
That seems like it should be a mistake in testing the speed.  You should be able to implement both the integer and floating-point functions without new bifs and by avoiding an extra function call the result should be faster if you test many iterations (to avoid GC skewing the results).